test/hardhat/helper.test.ts

```diff
@@ -0,0 +1,74 @@
+import { Helper } from '../../src/submodules/helper'
+const { parseProof, simulateVerify } = Helper
+import { ethers } from 'hardhat'
+import { Verifier } from '../typechain-types'
+import { assert } from 'ethers'
+import * as fs from 'fs'
+
+describe('ezkl', () => {
+  let verifier: Verifier
+  let proofPath: string = './test/data/test.pf'
+  let rpc_url: string = 'http://127.0.0.1:8545/'
+  beforeAll(async () => {
+    // Instantiate contract
+    verifier = (await ethers.deployContract('Verifier')) as any
```

Comment:
`any`'s suck. we really don't want to see these. Also should already be type `Verify` line 9